src/main/java/org/folio/bulkops/mapper/UserToUserDtoMapper.java

```diff
@@ -0,0 +1,7 @@
+package org.folio.bulkops.mapper;
+
+import org.folio.bulkops.domain.bean.User;
+
+public interface UserToUserDtoMapper {
+  org.folio.bulkops.domain.dto.User mapUserToUserDto(User user);
```

Comment:
Is it possible to add package org.folio.bulkops.domain.dto.User to imports?

Reply:
Two classes with the same name, one of them should contain full class name

Reply:
https://github.com/folio-org/mod-bulk-operations/pull/211/files#r1623905907